modules/Bio/EnsEMBL/Utils/IO/GFFSerializer.pm

```diff
@@ -60,25 +64,37 @@ my %strand_conversion = ( '1' => '+', '0' => '.', '-1' => '-');
 =head2 new
 
     Constructor
-    Arg [1]    : Ontology Adaptor
-    Arg [2]    : Optional File handle
-    Arg [3]    : Default source of the features. Defaults to .
-    
+    Arg [1]    : Optional File handle
+    Arg [2]    : Default source of the features. Defaults to .
+
     Returntype : Bio::EnsEMBL::Utils::IO::GFFSerializer
 
+    Warning    : If legacy mandatory 'Ontology Adaptor' first argument provided.
```

Comment:
Feels like there is a verb missing somewhere in this sentence.

Reply:
updated the message